app/logic/validator_score_v1_logic.rb

```diff
@@ -311,11 +313,33 @@ def assign_software_version_score
             validator.validator_score_v1.software_version = vah.software_version
           end
         end
+
+        this_software_version = validator.validator_score_v1.software_version
+
+        # Gather software_version stat
+        if validator.validator_score_v1.active_stake
+          if software_versions[this_software_version]
+            software_versions[this_software_version] += \
+              validator.validator_score_v1.active_stake
+          else
```

Comment:
You can initialize software_versions with `Hash.new(0)` to get rid of this if-else.